scripts/install_travis_proto.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Abort on error (-e) and print commands (-v).
+set -ev
+
+# See README.md in lnrpc why we need these specific versions/commits.
+PROTOC_VERSION=3.4.0
+PROTOBUF_VERSION="b5d812f8a3706043e23a9cd5babf2e5423744d30"
+GENPROTO_VERSION="a8101f21cf983e773d0c1133ebc5424792003214"
+GRPC_GATEWAY_VERSION="v1.8.6"
+
+# This script is specific to Travis CI so we only need to support linux x64.
+PROTOC_URL="https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip"
+PROTOC_DL_CACHE_DIR="${DOWNLOAD_CACHE:-/tmp/download_cache}/protoc"
+
+# install_protoc copies the cached protoc binary to the $PATH or downloads it
+# if no cached version is found.
+install_protoc() {
+  if [ -f "${PROTOC_DL_CACHE_DIR}/bin/protoc" ]; then
+    echo "Using cached version of protoc"
+  else
+    wget -O /tmp/protoc.zip $PROTOC_URL
+    mkdir -p "${PROTOC_DL_CACHE_DIR}"
+    unzip -o /tmp/protoc.zip -d "${PROTOC_DL_CACHE_DIR}"
+    chmod +x "${PROTOC_DL_CACHE_DIR}/bin/protoc"
+  fi
+  sudo cp "${PROTOC_DL_CACHE_DIR}/bin/protoc" /usr/local/bin
+  sudo cp -r "${PROTOC_DL_CACHE_DIR}/include" /usr/local
+  sudo chmod +r /usr/local/include -R
```

Comment:
Wouldn't a simple `cp -a` be enough`?

Reply:
No, I think this was required because the copy was done with `sudo` and in the original zip the "anyone can read" permission was missing. So only root could have read it.
But I fixed it in a slightly different way now.

Reply:
:+1: